internal/cmd/broker/register_broker_test.go

```diff
@@ -39,13 +32,13 @@ var _ = Describe("Register Broker Command test", func() {
 		command = NewRegisterBrokerCmd(context)
 	})
 
-	validRegisterBrokerExecution := func(args []string) *cobra.Command {
+	validAsyncRegisterBrokerExecution := func(args []string, location string) *cobra.Command {
```

Comment:
Minor: in all other similar methods to this one, the _location_ parameter always comes first. For consistency reasons, is it a good idea to switch the arguments here?